.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ workflows:
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.17
+      - image: cimg/go:1.19
```

Comment:
It seems previous `circleci/` images were deprecated some time ago.
https://circleci.com/docs/circleci-images/